### DISCUSSION TITLE
email to sms

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2002] [Help with using Postal]
One quick and honest question please, is it possible to use postal smtp to send email to sms?? well, i tried to send 1000 email to sms (sprint) and after about the 600th , i started getting SOFT FAIL THAT SAYS SENDER REJECTED, is there any way to go about it or fix it?? or is there another way to send sms with email in bulk??. kindly share your idea with me if you have any , thanks in advance.

**Comment by willpower232** [comment]
Postal can send email but you'd have to find a service that accepted email and sent the SMS messages. If you're having problems with something that is converting email to SMS then you should take it up with them.
